diff --git a/plasTeX/__init__.py b/plasTeX/__init__.py
index 2e4529d..f4ababc 100644
--- a/plasTeX/__init__.py
+++ b/plasTeX/__init__.py
@@ -1594,6 +1594,8 @@ class TheCounter(Command):
         # In this case, whenever a counter value comes out as a zero,
         # just hank it out.  This is especially useful in document classes
         # such as book and report which do this in the \thefigure format macro.
-        t = re.sub(r'\b0[^\dA-Za-z]+', r'', t)
+
+        # REMARK: disabled this for Gerby
+        #t = re.sub(r'\b0[^\dA-Za-z]+', r'', t)
 
         return tex.textTokens(t)
