diff --git a/plasTeX/Base/LaTeX/Math.py b/plasTeX/Base/LaTeX/Math.py
index 9e215e1..40d2160 100644
--- a/plasTeX/Base/LaTeX/Math.py
+++ b/plasTeX/Base/LaTeX/Math.py
@@ -6,7 +6,7 @@ C.7 Mathematical Formulas (p187)
 """
 
 from plasTeX.Base.LaTeX.Arrays import Array
-from plasTeX import Command, NoCharSubEnvironment, sourceChildren
+from plasTeX import Command, Environment, sourceChildren
 from plasTeX import DimenCommand, GlueCommand
 from plasTeX.Logging import getLogger
 
@@ -36,8 +36,9 @@ class ThinSpace_(Command):
     macroName = '/'
     str = '\u2009'
 
-class MathEnvironment(NoCharSubEnvironment):
+class MathEnvironment(Environment):
     mathMode = True
+    doCharSubs = False
 
 class MathEnvironmentPre(MathEnvironment):
     """
@@ -226,6 +227,7 @@ class ddots(Command):
 #
 
 class MathSymbol(Command):
+    doCharSubs = False
     pass
 
 # Lowercase
