diff --git a/plasTeX/Renderers/Gerby/Crossref.jinja2s b/plasTeX/Renderers/Gerby/Crossref.jinja2s
index b02c1f0..16e4b9f 100644
--- a/plasTeX/Renderers/Gerby/Crossref.jinja2s
+++ b/plasTeX/Renderers/Gerby/Crossref.jinja2s
@@ -1,7 +1,4 @@
 name: ref
-<a href="{{here.idref.label.url}}">{{here.idref.label.ref}}</a>
-
-name: pageref
-<a href="{{ obj.idref.label.url }}">*</a>
+<a href="/tag/{{ here.idref.label.userdata['tag'] }}">{{ here.idref.label.tagName|capitalize }} {{ here.idref.label.ref }}</a>
 
 name: label
