diff --git a/plasTeX/Renderers/Gerby/Crossref.jinja2s b/plasTeX/Renderers/Gerby/Crossref.jinja2s
index a5a17b6..779a340 100644
--- a/plasTeX/Renderers/Gerby/Crossref.jinja2s
+++ b/plasTeX/Renderers/Gerby/Crossref.jinja2s
@@ -2,3 +2,6 @@ name: ref
 <a href="/tag/{{ here.idref.label.userdata['tag'] }}" data-tag="{{ here.idref.label.userdata['tag'] }}">{{ here.idref.label.ref }}</a>
 
 name: label
+
+name: url
+<a href="{{ obj.attributes.url or obj }}">{{ obj.attributes.url or obj }}</a>
