diff --git a/plasTeX/Base/LaTeX/Quotations.py b/plasTeX/Base/LaTeX/Quotations.py
index 30c6f01..70ca649 100644
--- a/plasTeX/Base/LaTeX/Quotations.py
+++ b/plasTeX/Base/LaTeX/Quotations.py
@@ -21,3 +21,6 @@ class verse(Environment):
 # TODO figure out why putting this in plasTeX.Renderers.Gerby causes things to malfunction
 class slogan(Environment):
   blockType = True
+
+class history(Environment):
+  blockType = True
