diff --git a/plasTeX/__init__.py b/plasTeX/__init__.py
index f6df3db..fbfec38 100644
--- a/plasTeX/__init__.py
+++ b/plasTeX/__init__.py
@@ -928,16 +928,9 @@ class NoCharSubEnvironment(Environment):
         self.charsubs = []
         super(NoCharSubEnvironment, self).__init__(*args, **kwargs)
 
-    def invoke(self, tex):
-        # The goal is to prevent any character substitution while handling a
-        # this environment.
-        doc = self.ownerDocument
-        if self.macroMode == Macro.MODE_BEGIN:
-            self.charsubs = doc.charsubs
-            doc.charsubs = []
-        elif self.macroMode == Macro.MODE_END:
-            doc.charsubs = self.charsubs
-        super(NoCharSubEnvironment, self).invoke(tex)
+    def normalize(self, charsubs=None):
+        """ Normalize, but don't allow character substitutions """
+        return Environment.normalize(self)
 
 class NoCharSubCommand(Command):
     """
