diff --git a/plasTeX/Renderers/Gerby/__init__.py b/plasTeX/Renderers/Gerby/__init__.py
index 2433207..399cbc9 100644
--- a/plasTeX/Renderers/Gerby/__init__.py
+++ b/plasTeX/Renderers/Gerby/__init__.py
@@ -133,6 +133,9 @@ class Gerby(_Renderer):
     rendererdata = document.rendererdata["gerby"] = dict()
     config = document.config
 
+    # split-level must always be -2 to not create any clutter files
+    config["files"]["split-level"] = -2
+
     rendererDir = os.path.dirname(__file__)
 
     srcDir = document.userdata['working-dir']
