diff --git a/plasTeX/__init__.py b/plasTeX/__init__.py
index b298fbd..2e4529d 100644
--- a/plasTeX/__init__.py
+++ b/plasTeX/__init__.py
@@ -608,6 +608,10 @@ class Macro(Element):
                 macroargs.append(Argument('*modifier*', index, {'spec':item}))
                 index += 1
 
+            # Do not strip leading whitespace
+            elif item in '!':
+                argdict['stripLeadingWhitespace'] = False
+
             # Optional equals
             elif item in '=':
                 argdict.clear()
