diff --git a/plasTeX/Renderers/Gerby/__init__.py b/plasTeX/Renderers/Gerby/__init__.py
index 7b1e33a..8697ad5 100644
--- a/plasTeX/Renderers/Gerby/__init__.py
+++ b/plasTeX/Renderers/Gerby/__init__.py
@@ -16,8 +16,6 @@ from plasTeX.Renderers import Renderable, mixin, unmix
 from plasTeX.DOM import Node
 import json
 
-
-import pdb
 log = plasTeX.Logging.getLogger()
 
 
@@ -150,8 +148,8 @@ def linearRepresentation(document):
       document.userdata["linear"].append(node)
 
     stack.extend(node.childNodes)
-  pdb.set_trace()
-  document.userdata["linear"] = list(reversed(document.userdata["linear"]))
+
+  document.userdata["linear"] = list(reversed(document.userdata["linear"])) # TODO figure out why this is necessary
 
 def partsList(document):
   """Make an association between parts and chapters"""
