diff --git a/plasTeX/Renderers/Gerby/Lists.jinja2s b/plasTeX/Renderers/Gerby/Lists.jinja2s
index e42a469..08b1a9d 100644
--- a/plasTeX/Renderers/Gerby/Lists.jinja2s
+++ b/plasTeX/Renderers/Gerby/Lists.jinja2s
@@ -17,5 +17,12 @@ name: description
 </dl>
 
 name: item
-{% if obj.attributes['term'] %}<li class="custom-item" data-marker="{{ obj.attributes['term'] }}">{% else %}<li>{% endif %}{{ obj }}</li>
-{# TODO handle labels, tags #}
+{% if obj.attributes['term'] %}
+<li class="custom-item" data-marker="{{- obj.attributes['term'] -}}">
+{% elif obj.userdata['tag'] %}
+<li class="custom-item" data-marker="{{- obj.userdata['tag'] -}}" id="{{- obj.userdata['tag'] -}}">
+{% else %}
+<li>
+{% endif %}
+  {{- obj -}}
+</li>
