diff --git a/plasTeX/Renderers/Gerby/Misc.jinja2s b/plasTeX/Renderers/Gerby/Misc.jinja2s
index be6e8ba..6ace6cc 100644
--- a/plasTeX/Renderers/Gerby/Misc.jinja2s
+++ b/plasTeX/Renderers/Gerby/Misc.jinja2s
@@ -13,7 +13,7 @@ name: tableofcontents input @vobeyspaces @noligs def protect let \
       bibliography bibliographystyle
 
 name: texorpdfstring
-{{ obj.attributes.tex }}
+{{ obj.argSources['tex'][1:-1] }}
 
 name: href
 <a href="{{ obj.argSources['url'][1:-1] }}">{{ obj }}</a>
