diff --git a/plasTeX/Renderers/Gerby/Misc.jinja2s b/plasTeX/Renderers/Gerby/Misc.jinja2s
index 6ace6cc..be6e8ba 100644
--- a/plasTeX/Renderers/Gerby/Misc.jinja2s
+++ b/plasTeX/Renderers/Gerby/Misc.jinja2s
@@ -13,7 +13,7 @@ name: tableofcontents input @vobeyspaces @noligs def protect let \
       bibliography bibliographystyle
 
 name: texorpdfstring
-{{ obj.argSources['tex'][1:-1] }}
+{{ obj.attributes.tex }}
 
 name: href
 <a href="{{ obj.argSources['url'][1:-1] }}">{{ obj }}</a>
