diff --git a/plasTeX/Packages/xy.py b/plasTeX/Packages/xy.py
index e582f23..e1c2964 100644
--- a/plasTeX/Packages/xy.py
+++ b/plasTeX/Packages/xy.py
@@ -69,3 +69,14 @@ class xymatrixL(xymatrix):
 
 class xymatrix1(xymatrix):
   macroName = "xymatrix@1"
+
+class xyoption(Command):
+  # Options for xymatrix; unless we are writing xymatrix completely in python,
+  # this is mostly irrelevant.
+  pass
+
+class useTwocells(Command):
+  macroName ="UseTwocells"
+
+class useAllTwocells(Command):
+  macroName = "UseAllTwocells"
