diff --git a/plasTeX/Packages/hyperref.py b/plasTeX/Packages/hyperref.py
index b87888e..822383b 100644
--- a/plasTeX/Packages/hyperref.py
+++ b/plasTeX/Packages/hyperref.py
@@ -105,16 +105,16 @@ class autoref(Command):
     args = 'label:idref'
 
 class pdfstringdef(Command):
-    args = 'macroname:string tex:string'
+    args = 'macroname:str tex:str'
 
 class texorpdfstring(Command):
-    args = 'tex:string pdf:string'
+    args = 'tex:str pdf:str'
 
 class pdfstringdefDisableCommands(Command):
-    args = 'tex:string'
+    args = 'tex:str'
 
 class hypercalcbp(Command):
-    args = 'size:string'
+    args = 'size:str'
 
 
 # Forms
