diff --git a/plasTeX/Base/LaTeX/Math.py b/plasTeX/Base/LaTeX/Math.py
index 2709224..9e215e1 100644
--- a/plasTeX/Base/LaTeX/Math.py
+++ b/plasTeX/Base/LaTeX/Math.py
@@ -6,7 +6,7 @@ C.7 Mathematical Formulas (p187)
 """
 
 from plasTeX.Base.LaTeX.Arrays import Array
-from plasTeX import Command, Environment, sourceChildren
+from plasTeX import Command, NoCharSubEnvironment, sourceChildren
 from plasTeX import DimenCommand, GlueCommand
 from plasTeX.Logging import getLogger
 
@@ -36,7 +36,7 @@ class ThinSpace_(Command):
     macroName = '/'
     str = '\u2009'
 
-class MathEnvironment(Environment):
+class MathEnvironment(NoCharSubEnvironment):
     mathMode = True
 
 class MathEnvironmentPre(MathEnvironment):
