diff --git a/plasTeX/Renderers/HTML5/Misc.jinja2s b/plasTeX/Renderers/HTML5/Misc.jinja2s
index 23224c2..4282e80 100755
--- a/plasTeX/Renderers/HTML5/Misc.jinja2s
+++ b/plasTeX/Renderers/HTML5/Misc.jinja2s
@@ -7,4 +7,5 @@ name: tableofcontents input @vobeyspaces @noligs def protect let \
       newcommand hfill hline openout renewcommand write hspace appendix \
       global noindent include markboth setcounter refstepcounter \
       medskip smallskip parindent indent setlength settowidth addtolength \
-      nopagebreak newwrite newcounter typeout sloppypar hfil thispagestyle vfill
+      nopagebreak newwrite newcounter typeout sloppypar hfil thispagestyle vfill \
+      hypertarget reversemarginpar marginnote
