diff --git a/plasTeX/Packages/book.py b/plasTeX/Packages/book.py
index f353e11..007b376 100644
--- a/plasTeX/Packages/book.py
+++ b/plasTeX/Packages/book.py
@@ -13,7 +13,7 @@ def ProcessOptions(options, document):
 
     # Sections
     context.newcounter('part', resetby='volume',
-                       format='${part.Roman}')
+                       format='$part')
     context.newcounter('chapter', resetby='volume',
                        format='$chapter')
     context.newcounter('section', resetby='chapter',
