diff --git a/plasTeX/Tokenizer.py b/plasTeX/Tokenizer.py
index 98715eb..d9d5c85 100644
--- a/plasTeX/Tokenizer.py
+++ b/plasTeX/Tokenizer.py
@@ -30,6 +30,9 @@ DEFAULT_CATEGORIES = [
 VERBATIM_CATEGORIES = [''] * 16
 VERBATIM_CATEGORIES[11] = encoding.stringletters()
 
+class EndInput(Exception):
+    pass
+
 class Token(Text):
     """ Base class for all TeX tokens """
 
@@ -383,7 +386,10 @@ class Tokenizer(object):
                 yield mybuffer.pop(0)
 
             # Get the next character
-            token = next(charIter)
+            try:
+                token = next(charIter)
+            except StopIteration:
+                raise EndInput
 
             if token.nodeType == ELEMENT_NODE:
                 raise ValueError('Expanded tokens should never make it here')
