diff --git a/plasTeX/Renderers/Gerby/Thms.jinja2s b/plasTeX/Renderers/Gerby/Thms.jinja2s
index f456e16..730a303 100644
--- a/plasTeX/Renderers/Gerby/Thms.jinja2s
+++ b/plasTeX/Renderers/Gerby/Thms.jinja2s
@@ -1,5 +1,5 @@
 name: thmenv
-<article class="env-{{ obj.thmName }}" id="{{ obj.id }}">
+<article class="env-{{ obj.thmName }}" id="{{ obj.tag }}">
   {% if obj.title %}
     <p><a href="#"><span class="environment-identifier">{{ obj.thmName|capitalize }} {{ obj.ref }} ({{ obj.title }})</a>
   {% else %}
