diff --git a/plasTeX/Renderers/Gerby/__init__.py b/plasTeX/Renderers/Gerby/__init__.py
index ea82f7e..cee4fbe 100644
--- a/plasTeX/Renderers/Gerby/__init__.py
+++ b/plasTeX/Renderers/Gerby/__init__.py
@@ -239,6 +239,9 @@ class Gerby(_Renderer):
     # remove empty paragraphs
     s = re.compile(r'<p>\s*</p>', re.I).sub(r'', s)
 
+    # Remove extra paragraph tags around displaymath
+    s = re.compile(r'<p>(<div class="equation">.*?<\/div>)<\/p>', flags=re.DOTALL).sub(r'\1',s)
+
     return s
 
   def render(self, document):
