diff --git a/plasTeX/Renderers/HTML5/Config.py b/plasTeX/Renderers/HTML5/Config.py
index 58347da..27ada0c 100644
--- a/plasTeX/Renderers/HTML5/Config.py
+++ b/plasTeX/Renderers/HTML5/Config.py
@@ -91,3 +91,31 @@ section['filters'] = MultiOption(
     category='html5',
     default='',
 )
+
+section['tikz-compiler'] = StringOption(
+    """ LaTeX compiler for TikZ pictures """,
+    options='--tikz-compiler',
+    category='html5',
+    default='pdflatex',
+)
+
+section['tikz-converter'] = StringOption(
+    """ PDF to SVG converter for tikz and tikz-cd """,
+    options='--tikz-converter',
+    category='html5',
+    default='pdf2svg',
+)
+
+section['tikz-template'] = StringOption(
+    """ Jinja2 template file for tikz """,
+    options='--tikz-template',
+    category='html5',
+    default='',
+)
+
+section['tikz-cd-template'] = StringOption(
+    """ Jinja2 template file for tikz-cd """,
+    options='--tikz-cd-template',
+    category='html5',
+    default='',
+)
