diff --git a/plasTeX/Renderers/Gerby/Bibliography.jinja2s b/plasTeX/Renderers/Gerby/Bibliography.jinja2s
index b4c52bf..d2ef0f9 100644
--- a/plasTeX/Renderers/Gerby/Bibliography.jinja2s
+++ b/plasTeX/Renderers/Gerby/Bibliography.jinja2s
@@ -1,13 +1,8 @@
 name: cite
 <span class="cite">
 [
-{# TODO this needs attention: plasTeX needs to be aware of the BibTeX for it to work? #}
-{% for item in obj.citation() %}
-  {% if item.idref and item.idref.bibitem.url %}
-    <cite><a href="{{ item.idref.bibitem.url }}">{{ item }}</a></cite>
-  {% else %}
-    {{ item }}
-  {% endif %}
+{% for item in obj.attributes['bibkeys'] %}
+  <a href="/bibliography/{{ item }}">{{ item }}</a>
 {% endfor %}
 ]
 </span>
