diff --git a/plasTeX/Renderers/Gerby/Config.py b/plasTeX/Renderers/Gerby/Config.py
index df7104a..315e857 100644
--- a/plasTeX/Renderers/Gerby/Config.py
+++ b/plasTeX/Renderers/Gerby/Config.py
@@ -13,3 +13,31 @@ section["tags"] = StringOption(
   category = "gerby",
   default = "tags",
 )
+
+section['tikz-compiler'] = StringOption(
+    """ LaTeX compiler for TikZ pictures """,
+    options='--tikz-compiler',
+    category='gerby',
+    default='pdflatex',
+)
+
+section['tikz-converter'] = StringOption(
+    """ PDF to SVG converter for tikz and tikz-cd """,
+    options='--tikz-converter',
+    category='gerby',
+    default='pdf2svg',
+)
+
+section['tikz-template'] = StringOption(
+    """ Jinja2 template file for tikz """,
+    options='--tikz-template',
+    category='gerby',
+    default='',
+)
+
+section['tikz-cd-template'] = StringOption(
+    """ Jinja2 template file for tikz-cd """,
+    options='--tikz-cd-template',
+    category='gerby',
+    default='',
+)
