--- a/plasTeX/Base/LaTeX/Accents.py	2024-03-07 09:47:38.426386198 +0000
+++ b/plasTeX/Base/LaTeX/Accents.py	2024-03-07 13:18:38.437512799 +0000
@@ -30,6 +30,14 @@
                     content[0]+self.middle_combining+content[1:]+self.combining)
 
     @property
+    def source(self):
+        textSrc = self.textContent.strip()
+        theChars = type(self).chars
+        if textSrc in theChars :
+          return theChars[textSrc]
+        return super().source
+
+    @property
     def textContent(self):
         """
         We need a customized textContent that doesn't look up

