--- a/plasTeX/Base/LaTeX/Accents.py
+++ b/plasTeX/Base/LaTeX/Accents.py
@@ -30,6 +30,10 @@
                     content[0]+self.middle_combining+content[1:]+self.combining)

     @property
+    def source(self):
+        return type(self).chars.get(self.textContent.strip(), None)
+
+    @property
     def textContent(self):
         """
         We need a customized textContent that doesn't look up
