diff --git a/plasTeX/Renderers/Gerby/Crossref.jinja2s b/plasTeX/Renderers/Gerby/Crossref.jinja2s
index df1d2fa..a5a17b6 100644
--- a/plasTeX/Renderers/Gerby/Crossref.jinja2s
+++ b/plasTeX/Renderers/Gerby/Crossref.jinja2s
@@ -1,4 +1,4 @@
 name: ref
-<a href="/tag/{{ here.idref.label.userdata['tag'] }}">{{ here.idref.label.ref }}</a>
+<a href="/tag/{{ here.idref.label.userdata['tag'] }}" data-tag="{{ here.idref.label.userdata['tag'] }}">{{ here.idref.label.ref }}</a>
 
 name: label
